### PR TITLE
create section-1

### PR DESCRIPTION
**Pull request by alexdum123**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -19,4 +19,56 @@ body {
...
+    justify-content: center;
+    align-items: center;
+    width: 3.5vw;
+    height: 3.5vw;
+    border-radius: 5px;
+    transition: all 0.3s ease;
+    position: absolute;
+    z-index: 1;
+    transform: translate(0.65vw, -4.3vw);
+
```
Comment (SozonikOleg) [inline css/style.css]: Remove space

File: index.html [hunk 2]
```diff
@@ -131,6 +131,73 @@
...
+                                stroke-linejoin="round" />
+                            <path d="M12 5L19 12L12 19" stroke="white" stroke-width="2" stroke-linecap="round"
+                                stroke-linejoin="round" />
+                        </svg>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </main>
+    <br>
```
Comment (SozonikOleg) [inline index.html]: Improve this code.
Delete a lot br.

File: script.js [hunk 3]
```diff
@@ -1,6 +1,20 @@
-function performSearch() {
-    var searchTerm = document.getElementById('search-input').value;
+// function performSearch() {
+//     var searchTerm = document.getElementById('search-input').value;
+  
+//     var searchResults = document.getElementById('search-results');
+//     searchResults.innerHTML = '<p>Результаты поиска для: ' + searchTerm + '</p>';
+// }
```
Comment (SozonikOleg) [inline script.js]: Delete this comment

**Review by SozonikOleg — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (alexdum123) [reply index.html]: Tested the layout, forgot to remove

**Review by SozonikOleg — APPROVED** [review]

**Pull request merged**